src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {
+            tabs.map(tab => (
+              <li
+                key={tab.id}
+                className={
+                  classNames(null, {
```

Comment:
First param redundant
```suggestion
                  classNames({
```